include/odp/api/spec/crypto.h

```diff
@@ -647,7 +647,8 @@ int odp_crypto_auth_capability(odp_auth_alg_t auth,
  *
  * Create a crypto session according to the session parameters. Use
  * odp_crypto_session_param_init() to initialize parameters into their
- * default values.
+ * default values. If call ends up with an error, 'session' argument won't be
+ * updated and no new session will be created.
```

Comment:
I'm not sure how this is a clarification. The documentation for the session parameter below clearly states it is set to the created session or ODP_CRYPTO_SESSION_INVALID, so in either case it's an output parameter and is set by this API.